charts/grafana/README.md

```diff
@@ -53,6 +53,11 @@ to `global.imageRegistry`. If you were not previously setting `global.image.regi
 is required on upgrade. If you were previously setting `global.image.registry`, you will
 need to instead set `global.imageRegistry`.
 
+### To 10.0.0
+
+Static alerting resources now support Helm templating. If you are using the `alerting` configuration,
+double curly braces must be escaped using `{{ "{{" }}` syntax to prevent unintended template processing.
```

Comment:
```suggestion
Static alerting resources now support Helm templating. This means that alerting resources loaded from external files (`alerting.*.files`) are now processed by the Helm template engine.

If you already use template expressions intended for Alertmanager (for example, `{{ $labels.instance }}`), these must now be escaped to avoid unintended Helm evaluation. To escape them, wrap the braces with an extra layer like this:

`{{ "{{" }} $labels.instance {{ "}}" }}`

This ensures the expressions are preserved for Alertmanager instead of being rendered by Helm.
```